edifact-edi/2.2/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,107 @@
+
+The message contains the following keys, some of which apply to either the Read operation or Write operation, as indicated:
+
+[%header%autowidth.spread]
+|===
+|Key Name |Description
+|Delimiters |An optional set of delimiters used in the message. If used in a read operation, the delimiters are based on the most recent interchange processed by a read operation. If used for a write operation, the delimiters override the values in the module configuration. The characters in the string of delimiters are interpreted based on their position in string, in the following order:
+data element separator, component element separator, repetition separator, segment terminator, release character.
+|Errors (read only) |A list of errors that are associated either with the input as a whole or with interchanges that have no valid transaction sets.
+|FunctionalAcksGenerated (read only) |A list of `CONTRL` acknowledgments that are generated by the module during the read operation.
+|FunctionalAcksToSend (write only) |A list of `CONTRL` acknowledgments that are sent by the module during the write operation.
+|Interchange (write only) |Map of UNB interchange header segment values used as defaults during an interchange write operation.
+|Messages |A hierarchy of messages that are either read by the module during a read operation or sent by the module during a write operation. The value of this top-level key is a map with standard versions as keys, in the form `D96A`. For example, if you are using version `D96.A ORDERS` and `CUSRES` messages, the Messages contain a map with one key, `D96A`. The value of this key is another map, one with two keys, `ORDERS` and `CUSRES`. Each of these contains a list of individual ORDERS and CUSRES messages that are processed.
+|===
+
+Individual messages have their own maps, with the following keys:
+
+[%header%autowidth.spread]
+|===
+|Key |Description
+|Detail |Map of segments or loops from the detail section of the message. Values are maps for segments or loops that occur once. Lists of maps are for values that occur more than once.
+|Errors (read only) |A list of errors associated with the message.
+|Heading |A map of segments or loops from the heading section of the message. Values are maps for segments or loops that occur once. Lists of maps exist for values that occur more than once.
+|Id |The Message ID must match the key of the containing message list.
+|Interchange |A map of UNB interchange header segment values. For a read message, the map is the actual data from the enclosing interchange. It is a single map linked from all messages in the interchange. For a write message, the map value creates the enclosing interchange. For example, gathering messages with the same interchange values into a single interchange, regardless of whether the actual maps are the same. If there are no values for this map, then the default is the value(s) taken from the Interchange map at the message level.
```

Comment:
Replace value(s) with the most likely of value or values; never use this (s) structure.